### DISCUSSION TITLE
Companies & Developers Using Spark

### DISCUSSION THREAD

**Discussion by westonwalker** [spark-dotnet/framework#10] [Announcements]
If you are using Blazor Spark personally or at your company, please share the URL here!

If you don't have a public url available, share a screen shot and the name of the app.

I'll be building a showcase page on blazorspark.com

**Comment by westonwalker** [comment]
A job board for .NET developers.

https://dotnet-jobs.com/

**Comment by trifleneurotic** [comment]
A hospitality web app for viewing, cycling and generating codes for properties that use keypad locks for rooms/areas (instead of traditional coded keys).

<img width="820" alt="RoomCoder" src="https://github.com/spark-dotnet/framework/assets/1438360/19dcb500-f36c-48fd-85f3-f5836285345f">
